### PR TITLE
send telemetry when 'skip' is clicked in alternative flow

### PR DESCRIPTION
**Pull request by shakyShane**
### TL;DR
This change adds a new telemetry event ('toggleSkipped') for when users skip the toggle step on the choice-category screen.

### What changed?
1. Now sends the 'toggleSkipped' event via the existing telemetry handler
2. Adjusted integration tests to check for the new telemetry event.
3. Updated TypeScript interfaces to include the new event.

### How to test (during native integration)
1. Navigate to the choice-category screen.
2. Click on the 'Skip this step' link.
3. Verify that the telemetry event 'toggleSkipped' is sent.
4. Check the modified integration tests for the new event validation.

---
new breakage flows

ScreenFlow

**Reviewer:**

### CODE REVIEW

**Comment by shakyShane** [comment]
* **#158** <a href="https://app.graphite.dev/github/pr/duckduckgo/privacy-dashboard/158?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @shakyShane and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by jaceklyp — APPROVED** [review]
tested on my end, it works, thank you!
